modules/ROOT/pages/java-se.adoc

```diff
@@ -10,19 +10,19 @@
 :page-type: general
 = Java SE support
 
-Open Liberty requires a Java SE runtime. A Java SE runtime, also known as a Java Runtime Environment or a Java Developer Kit (JDK), provides the tools to develop, run, and monitor your Java application.
+Open Liberty requires a Java SE runtime. A Java SE runtime, also known as a Java Runtime Environment or a Java Developer Kit (JDK), provides the tools to develop, run, and monitor your Java application. Open Liberty runs on any of the Java runtimes listed in the following sections. 
```

Comment:
Suggested update

>Open Liberty runs on any of the Java runtimes listed in the following sections. 

To

>Open Liberty runs on any of the Java versions that are listed in the following sections.

- Does _versions_ work here?

Reply:
That works fine.  I will make that change and push out the update file.